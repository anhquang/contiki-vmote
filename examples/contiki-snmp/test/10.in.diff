1,5c1,2
< SNMPv2-MIB::system.11.0 = STRING: "Pointer to a string"
< in packet
< Reason: (noSuchName) There is no such variable name in this MIB.
< Failed object: iso.4
< 
---
> getaddrinfo:  No address associated with hostname
> Timeout: No Response from udp6:[aaaa::fdff:ffff:ffff:ffff].
