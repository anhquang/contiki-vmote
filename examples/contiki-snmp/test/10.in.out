getaddrinfo:  No address associated with hostname
Timeout: No Response from udp6:[aaaa::fdff:ffff:ffff:ffff].
